scripts/delayed.js

```diff
@@ -5,3 +5,9 @@ import { sampleRUM } from './aem.js';
 sampleRUM('cwv');
 
 // add more delayed functionality here
+
+// Launch script
+const script = document.createElement('script');
+script.src = 'https://assets.adobedtm.com/launch-ENd3e1f5f2e5b44e4f8c9e7b0c4a2f8b3.min.js';
```

Comment:
Can you use loadScript instead? The code will be much simpler for it.

Reply:
Done. Appreciate the feedback.